src/pages/Home.jsx

```diff
@@ -24,6 +24,9 @@ const Home = () => {
   };
 
   const renderApiData = () => {
+    if(isFetching){
+      return(<TextCenter>Data is Loading</TextCenter>)
```

Comment:
# Improve 
- This PR can be easily improved by adding few changes
- add a spinner or well styled Text (animated is prefferable ) but atleast add some style.
- after changes add working Screnshot's of the  change in the PR